### ISSUE TITLE
Update Node 16 -> 20

### ISSUE THREAD

**Issue by kagankan** [MishaKav/jest-coverage-comment#79]
Thank you for creating such a fantastic tool! 😍 
I have been finding it extremely useful.

Since Node version 16 has become deprecated, could you please consider updating it to version 20?

see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/

GitHub shows such a warning.
![image](https://github.com/MishaKav/jest-coverage-comment/assets/34136752/90512539-13a5-4425-b1d2-c15c4918751b)

**Comment by MishaKav** [comment]
Sure, thanks for letting me know. 
will do it soon.

**Comment by MishaKav** [comment]
Thank you for your contribution. I merged your changes and release `v1.0.25`.

You can use the latest version 
```yaml
- name: Jest coverage comment
  uses: MishaKav/jest-coverage-comment@main
```
or 
```yaml
- name: Jest coverage comment
  uses: MishaKav/jest-coverage-comment@v1.0.25
```
